### PR TITLE
Add Router and all routes to pages

### LINKED ISSUE #11: Routes available in App
If we check the Design (#2) we can note these routes:

## `home` or root (`/`)

- [x] #42

image

## `check`

- [x] #48

image

## `delivery/:id`

- [x] #39 

image

# `chat/:id`

- [x] #40

image

Related issues
 
- #31
- #34 

## `confirm/:id`

- [x] #43

image

## error 404 and error 5XX

- [x] #49

image

- [ ] app error 5XX

TBD

Notice:
- Should include id and timestamp of error
- Should include developer/technical staff contact information
- Should have a link to Home or other TBD

## Checklist

- [x] Define the route links to access the views
- [x] A router package is added to manage routes
- [x] This package doesn't break the app
- [x] A user can change between routes using URL and using links
- [x] There is a fallback for a not found route
- [ ] There is a way to show errors 5xx

### PR DESCRIPTION
**Pull request by Israel-Laguan**
The idea here is to check if the router works as described in #11 

## How to test:

- [ ] Open [preview link](https://deploy-preview-64--azordev-dasher-user.netlify.app)
- [ ] test `home`, `check` and `error` routes
- [ ] Try to enter in `delivery`, `confirm` and `chat` and notice you are redirected to `check`
- [ ] Enter a not valid route and be redirected to 404 page
- [ ] Check the console for any error

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
Have to take into account `.htaccess` file  to use router in production

**Comment by Israel-Laguan** [comment]
I found an error while displaying `error` (LOL) 
image

**Comment by Israel-Laguan** [comment]
Tested and worked as expected

File: src/pages/Error.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+
+const Error = ({ errorInfo = { componentStack: [ 'error' ] } }) => (
```
Comment (Israel-Laguan) [inline src/pages/Error.jsx:3]: This default is for demonstration purposes

**Comment by AlejandroSalcido** [comment]
All the routes work as in the description, everything seems to be working as expected!

**Comment by denisosuna** [comment]
it works as expected

**Pull request merged**